File: app/src/main/java/com/artmurka/artmurkaapp/Views/Fragments/Interfaces/IWishFragment.java
package com.artmurka.artmurkaapp.Views.Fragments.Interfaces;

import com.artmurka.artmurkaapp.Model.Pojo.ItemList.WishList.GoodsListDescription;
import com.artmurka.artmurkaapp.Model.Pojo.ItemList.WishList.WishList;

import java.util.ArrayList;
import java.util.List;


public interface IWishFragment {
    void showWishList(List<GoodsListDescription> list );
}

File: app/src/main/java/com/artmurka/artmurkaapp/android/views/fragments/interfaces/IWishFragment.kt
package com.artmurka.artmurkaapp.android.views.fragments.interfaces

import com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList.GoodsListDescription
import com.artmurka.artmurkaapp.presenter.PresenterView


interface IWishFragment  : PresenterView {
    fun showWishList(list: MutableList<GoodsListDescription>)
    fun deleteFromWisList(goodId : String)
}
